mrtt-ui/src/data/questionMapping.js

```diff
@@ -1,9 +1,28 @@
+import questions from './questions'
+
 export const questionMapping = {
   projectDetails: {
-    hasProjectEndDate: '1.1a',
-    projectStartDate: '1.1b',
-    projectEndDate: '1.1c',
-    countries: '1.2',
-    siteArea: '1.3'
+    hasProjectEndDate: { apiQuestionId: '1.1a', questionType: 'singleChoice' },
+    projectStartDate: { apiQuestionId: '1.1b' },
+    projectEndDate: { apiQuestionId: '1.1c' },
+    countries: { apiQuestionId: '1.2' },
+    siteArea: { apiQuestionId: '1.3' }
+  },
+  restorationAims: {
+    ecologicalAims: {
+      apiQuestionId: '3.1',
+      questionType: 'multipleChoiceOther',
+      options: questions.ecologicalAims.options
+    },
+    socioEconomicAims: {
+      apiQuestionId: '3.2',
+      questionType: 'multipleChoiceOther',
+      options: questions.socioEconomicAims.options
+    },
+    otherAims: {
+      apiQuestionId: '3.3',
+      questionType: 'multipleChoiceOther',
+      options: questions.otherAims.options
+    }
```

Comment:
What do you think of this direction? Seemed like a nice place to put `questionType` and then it was convenient to attach the 'options'/answers array for easy reference when formatting multiple choice other questions (the code user the original options to compare what is an 'other'). At some point I need to check that this isnt a circular dependency. Should be ok because questions is a constant.